### PR TITLE
ENH: Better styling

### PR DESCRIPTION
**Pull request by larsoner**
Should wait for https://github.com/sphinx-gallery/sphinx-gallery/pull/752 to be merged then restore the config line, but this should allow you to check the outputs @rob-luke

### CODE REVIEW

**Comment by larsoner** [comment]
All better

https://594-254026779-gh.circle-artifacts.com/0/dev/auto_examples/plot_12_group_glm.html#compute-group-level-results

**Comment by rob-luke** [comment]
Massive thanks @larsoner! Those changes were way beyond me. Also thanks for taking a read of the tutorial and making those improvement to the text and docs.

File: requirements_doc.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 sphinx
 https://github.com/numpy/numpydoc/archive/master.zip
 sphinx_bootstrap_theme
-https://github.com/sphinx-gallery/sphinx-gallery/archive/master.zip
+https://github.com/larsoner/sphinx-gallery/archive/css.zip
```
Comment (larsoner) [inline requirements_doc.txt:4]: @rob-luke you can revert this line now that my `css` branch has been merged into the upstream `master`

Comment (rob-luke) [reply requirements_doc.txt:4]: Done in #184 
Thanks again

**Pull request merged**